python/caffe/test/test_python_layer.py

```diff
@@ -140,3 +165,14 @@ def test_parameter(self):
         self.assertEqual(layer.blobs[0].data[0], 1)
 
         os.remove(net_file)
+
+    def test_phase(self):
+        net_file = phase_net_file()
+        # Test on phase TRAIN
```

Comment:
You can drop these comments; they don't add anything to the code.

(And if you like, make this block a short loop: `for phase in caffe.TRAIN, caffe.TEST:`...)